Berlin/BerOr1166.xml

```diff
@@ -0,0 +1,284 @@
+</msItem>
+
+<msItem xml:id="ms_i1.11">
+<locus from="52r" to="55v"/>
+<title type="complete" ref="LIT6443Sunday6"/>
+<textLang mainLang="gez"></textLang>
+</msItem>
+
+<msItem xml:id="ms_i1.12">
+<locus from="55v" to="60r"/>
+<title type="complete" ref="LIT2458Testam"/>
+<note>Indication for reading: <foreign xml:lang="gez">አመ፩፡ ለሴነ<gap reason="lost"/></foreign></note>
+<textLang mainLang="gez"></textLang>
+</msItem>
+
+<msItem xml:id="ms_i1.13">
+<locus from="60v" to="66r"/>
+<title type="complete" ref="LIT6444HomilyPilatos"/>
+<note>Indication for reading: <foreign xml:lang="gez">በዓርበ፡ ስቅለት፡ በ፫፡</foreign></note>
+<textLang mainLang="gez"></textLang>
+</msItem>
+
+<msItem xml:id="ms_i1.14">
+<locus from="66r" to="69v"/>
+<title type="incomplete" ref="LIT6441Kehda"/>
+<note>Incomplete at the end.</note>
+<textLang mainLang="gez"></textLang>
+</msItem>
+
+</msItem>
+
+</msContents>
+
+<physDesc>
+<objectDesc form="Codex">
+<supportDesc>
+
+  <support>
+      <material key="parchment"/>
+  </support>
+
+  <extent>
+      <measure unit="leaf">70</measure>
+      <measure unit="leaf" type="blank">2</measure>
+<dimensions type="outer" unit="mm">
+<height>350</height>
+<width>240</width>
+<depth>35</depth>
+</dimensions>
+
+</extent>
+
+
+<foliation>
+F. 1 and 70 are blank.
+<list>
+
+<item xml:id="fol1">
+European foliation in pencil in the upper right corner, from 63 to 139.
+</item>
+
+<item xml:id="fol2">
+European foliation in pencil in the space between the two columns, from 1 to 69. Many numbers have been erased or crossed out.
+This is the foliation used as reference in this record.
+</item>
+
+</list>
+</foliation>
+
+<condition key="good">Some losses in parchment, in a few cases (e. g. <locus from="40" to="49">f. 40-49</locus>) causing a loss of text.
+<locus target="#2">F. 2</locus> very darkened.</condition>
+
+</supportDesc>
+
+
+<layoutDesc>
+      <layout columns="2" writtenLines="31 33">
+      <ab type="pricking">Ruling is partly visible; pricking is visible.</ab>
+      </layout>
+</layoutDesc>
+
+</objectDesc>
+
+<handDesc>
+   <handNote xml:id="h1" script="Ethiopic">
+    <seg type="script">
+    <date notBefore="1300" notAfter="1499"/></seg>
+     <seg type="ink">Black, red.</seg>
+     <seg type="rubrication">Numerals, elements of the punctuation signs, indications for reading.</seg>
+     <desc>Careful, well trained hand.</desc>
+   </handNote>
+ </handDesc>
+
+<decoDesc>
+
+<decoNote xml:id="d1">
+<locus target="#6r #27v #38r #45v #48r #52r"/>
+<desc>Crux ansata.</desc>
+</decoNote>
+
+</decoDesc>
+
+<additions>
+<list>
+
+<item xml:id="e1">
+<locus target="#21r #32v #43r"/>
+<desc type="Supplication">The name of the donor appearing in the supplications has been erased.</desc>
+</item>
+
+<item xml:id="e2">
+<locus target="#69r"/>
+<desc type="StampExlibris">Library stamp.</desc>
+</item>
+
+<item xml:id="e3">
+<desc type="MixedNote">The manuscript originally formed one codex with <ref type="mss" corresp="BerOr1165"/>.</desc>
```

Comment:
Should you feel ambitious, this would be an opportunity to make use of the Syntaxe du Codex relations ;)
https://betamasaheft.eu/Guidelines/?id=syntaxe